source/rst/functions.md

```diff
@@ -250,8 +250,8 @@ Notes
 
 * We are passing the argument `U` as a string, which is why we write it as `'U'`.
 * Notice that equality is tested with the `==` syntax, not `=`.
-    * > For example, the statement `> a = 10`>  assigns the name `> a`>  to the value `> 10`> .
-    * > The expression `> a == 10`>  evaluates to either `> True`>  or `> False`> , depending on the value of `> a`> .
+    * > For example, the statement `> a = 10`  assigns the name `> a` to the value `> 10`.
```

Comment:
the `>` should probably be removed which is causing a comment or note like admonition in markdown syntax

Reply:
The source `rst` is:

```rst
* Notice that equality is tested with the ``==`` syntax, not ``=``.

    * For example, the statement ``a = 10`` assigns the name ``a`` to the value ``10``.

    * The expression ``a == 10`` evaluates to either ``True`` or ``False``, depending on the value of ``a``.
```